### PR TITLE
adding list of local models as new argument for Ensemble constructor

### PR DESCRIPTION
**Pull request by mmerce**

### CODE REVIEW

File: bigml/ensemble.py [hunk 1]
```diff
@@ -67,25 +76,34 @@ def __init__(self, ensemble, api=None, max_models=None):
...
+                models = ensemble
+                self.model_ids = [local_model.resource_id for local_model in
+                                  models]
+            else:
+                try:
+                    models = [get_model_id(model) for model in ensemble]
+                    self.model_ids = models
+                except ValueError:
+                    raise ValueError('Failed to verify the list of models. Check '
+                                     'your model id values.')
```
Comment (jaor) [inline bigml/ensemble.py]: perhaps you could append here the message of the original exception, to help in diagnosing what's the wrong value...

File: bigml/ensemble.py [hunk 2]
```diff
@@ -280,8 +301,27 @@ def all_model_fields(self):
...
+                local_model = model_id
+            else:
+                local_model = Model(model_id, self.api)
             gc.collect()
             fields.update(local_model.fields)
-        return fields
+            if (objective_id is not None and
+                    objective_id != local_model.objective_id):
+                # the models' objective field have different ids, no global id
+                no_objective_id = True
```
Comment (jaor) [inline bigml/ensemble.py]: why don't you just assign here `objective_id=None` and do without the auxiliary variable `no_objective_id`? i think it'd be easier to follow...

Comment (mmerce) [inline bigml/ensemble.py]: ok

File: bigml/ensemble.py (hunk 1 above)
Comment (mmerce) [inline bigml/ensemble.py]: ok

File: bigml/ensemble.py (hunk 2 above)
Comment (jaor) [inline bigml/ensemble.py]: ok

File: HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,12 @@
 History
 -------
 
+2.2.0 (2015-02-26)
+~~~~~~~~~~~~~~~~~~
+
+- Adding lists of local models as we argument for the local ensemble
```
Comment (jaor) [inline HISTORY.rst]: typo: as we? argument

**Pull request merged**